compiler/pash_compilation_server.py

```diff
@@ -321,19 +329,28 @@ def compile_and_add(self, compiled_script_file, var_file, input_ir_file):
         else:
             ## Wait if we have more pipelines running than our current limit
             self.wait_until_limit(config.pash_args.parallel_pipelines_limit)
-
+        
         if compile_success:
             response = server_util.success_response(
                 f"{process_id} {compiled_script_file} {var_file} {input_ir_file}"
             )
+        elif not all_region_parallelizable: 
+            # send specified message to say not all regions are parallelizable instead of general exception caught
+            response = server_util.error_response(f"{process_id} not all regions are parallelizable; failed to compile")
```

Comment:
The response can be more specific, and slightly reworded. At this point we know precisely which region is not parallelizable, and we can respond with that (instead of a more generic "not all regions"). Also, the `all_region_parallelizable` flag could be renamed to `current_region_parallelizable` or `parallelizability_success` since it refers to a specific region and not all regions.

Reply:
would it be okay to response with "current region" along with the `process_id`? I'm not entirely sure how to refer to a region 
Thank you for the reviews!